src/components/TokenForm.js

```diff
@@ -35,15 +35,14 @@ const TokenForm = ({ setToken }) => {
               value={tokenInput}
               onChange={(event) => {
                 setTokenInput(event.target.value);
+                setError(false);
               }}
               required
             />
           </label>
           <button type="submit">Join an existing list</button>
         </form>
-        {hidden ? (
-          ''
-        ) : (
+        {error && (
```

Comment:
Chris also suggested to use an inline boolean guard to do conditional rendering instead of using a ternary